### PR TITLE
Fixed mime type listener reference

### PR DESCRIPTION
**Pull request by yceruto**
Fixed issue #1304

### CODE REVIEW

**Comment by Ener-Getick** [comment]
In fact this link must be removed and we should advise to add the mime types thanks to [the ``FOSRestBundle`` mime type listener](http://symfony.com/doc/master/bundles/FOSRestBundle/3-listener-support.html#mime-type-listener).

File: Resources/doc/format_listener.rst [hunk 1]
```diff
@@ -86,7 +86,7 @@ in the controller action.
 
 Note that the format needs to either be supported by the ``Request`` class
 natively or it needs to be added as documented here or using the
-`mime type listener`_ explained in the Symfony documentation.
+:doc:`Mime Type Listener <3-listener-support>` explained in the Symfony documentation.
```
Comment (yceruto) [inline Resources/doc/format_listener.rst]: I'm not sure now be consistent with the documentation.

Comment (Ener-Getick) [inline Resources/doc/format_listener.rst]: IMO this should be transformed to:
```rst
Note that the format needs to be added using the :doc:`Mime Type Listener <3-listener-support>`.
```

ping @lsmith77

Comment (yceruto) [inline Resources/doc/format_listener.rst]: Updated!

**Comment by Ener-Getick** [comment]
:+1: for me

Status: Reviewed

**Comment by Ener-Getick** [comment]
Thank you @yceruto ☺

**Pull request merged**